src/DDoSX/RecordClient.php

```diff
@@ -95,4 +95,18 @@ public function update(Record $record)
                 return new Record($this->apiToFriendly($response->data, $this->requestMap));
             });
     }
+
+    /**
+     * @param $domainName
+     * @param $recordId
+     * @return Record
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getById($domainName, $recordId)
+    {
+        $response = $this->request("GET", 'v1/domains/' . $domainName . '/records/' . $recordId);
+        $body = $this->decodeJson($response->getBody()->getContents());
+
+        return new Record($this->apiToFriendly($body, $this->requestMap));
```

Comment:
`$body` should be `$body->data`

Reply:
request map also needs some extra fields adding:

```
original_ttl => originalTtl
original_priority => originalPriority
insert_timestamp => insertedAt
update_timestamp => updatedAt
```

This would also require updates to the Record class doc blocks

Reply:
I've updated it